includes/admin/class-wc-payments-admin.php

```diff
@@ -1126,51 +1124,6 @@ public function maybe_redirect_to_onboarding() {
 		$this->account->redirect_to_onboarding_welcome_page();
 	}
 
-	/**
-	 * Avoid WC Admin /payments/overview error page when the current account associated Stripe account is not valid.
-	 *
-	 * The errored page happens because we don't register a /payments/overview WC admin page when the Stripe account
-	 * is not valid and register only a /payments/connect top level menu page.
-	 *
-	 * Places around our plugin redirect merchants to the overview page by default (or using it for the Stripe KYC
-	 * return URL) leading to poor UX.
-	 * This is a safety net to prevent that from happening.
-	 *
-	 * @see self::add_payments_menu()
-	 */
-	public function maybe_redirect_overview_to_connect() {
```

Comment:
this function is not needed because this redirect is handled in `maybe_redirect_from_payments_admin_child_pages`, confirmed by manual testing